integration/tests/conftest.py

```diff
@@ -255,3 +255,11 @@ def event_caller_contract(web3_client, class_account) -> typing.Any:
         "common/EventCaller", "0.8.12", class_account
     )
     yield event_caller
+
+
+@pytest.fixture(scope="class")
+def event_caller_with_receipt(web3_client, class_account) -> typing.Any:
```

Comment:
You need receipt only in one test. I think it is better to deploy contract in test, without fixture

Reply:
Done